storage/cql_iface/cql_iface.py

```diff
@@ -105,10 +99,18 @@ def fill_empty_keys_with_None(keys_dict, data_model):
         data_model = {k: None for k in data_model.keys()}
         return {**data_model, **keys_dict}
 
-    def put_record(self, object_id: UUID, key_list: dict, value_list: dict) -> None:
+
+    def delete_persistent_object(self, object_id: UUID):
+        if not isinstance(object_id, UUID):
+            raise ValueError("The object_id is not an UUID")
         try:
-            UUID(str(object_id))
+            CqlCOMM.delete_data(object_id)
```

Comment:
If we capture all ValueErrors we hide other problems. Might be it would be better to return true on success and false otherwise?